tests/test_evotuning.py

```diff
@@ -13,47 +13,58 @@
 
 
 @pytest.fixture
-def params():
-    """Return randomly initialized params."""
-    init_fun, _ = mlstm64()
-    _, parameters = init_fun(key=PRNGKey(0), input_shape=(-1, 10))
-    return parameters
+def model():
+    """Return mlstm with randomly init. params"""
+    init_fun, apply_fun = mlstm64()
+    _, params = init_fun(rng=PRNGKey(0), input_shape=(-1, 26))
+    return apply_fun, params
 
 
-def test_evotune():
+def test_evotune(model):
     """Simple execution test for evotune."""
     seqs = ["MTN", "BDD"] * 5
     n_epochs_config = {"high": 1}
-    init_func, model_func = mlstm64()
-    _, params = init_func(PRNGKey(42), (-1, 10))
+
+    _, _ = evotune(
+        sequences=seqs,
+        model_func=model[0],
+        params=model[1],
+        n_trials=1,
+        n_epochs_config=n_epochs_config,
+    )
+    # now test using all defaults
     _, _ = evotune(
         sequences=seqs,
-        model_func=model_func,
-        params=params,
         n_trials=1,
         n_epochs_config=n_epochs_config,
     )
 
 
 @pytest.mark.parametrize("holdout_seqs", (["ASDV", None]))
 @pytest.mark.parametrize("batch_method", (["length", "random"]))
-def test_fit(holdout_seqs, batch_method):
+def test_fit(model, holdout_seqs, batch_method):
     """Execution test for ``jax_unirep.evotuning.fit``."""
     sequences = ["ASDFGHJKL", "ASDYGHTKW", "HSKS", "HSGL", "ER"]
 
-    key = PRNGKey(42)
+    tuned_params = fit(
+        model_func=model[0],
+        params=model[1],
```

Comment:
Ditto as above. You can resolve this one if you're feeling lazy and don't wanna change this minor nitpick.